src/Illuminate/Foundation/Configuration/ApplicationBuilder.php

```diff
@@ -210,15 +210,10 @@ protected function buildRoutingCallback(array|string|null $web,
                 });
             }
 
-            if (is_string($web) || is_array($web)) {
-                if (is_array($web)) {
-                    foreach ($web as $webRoute) {
-                        if (realpath($webRoute) !== false) {
-                            Route::middleware('web')->group($webRoute);
-                        }
-                    }
-                } else {
-                    Route::middleware('web')->group($web);
+            if ($web) {
+                $web = (array) $web;
+                foreach ($web as $webRoute) {
```

Comment:
You can use `Illuminate\Support\Arr::wrap($web)` instead of `(array) $web`

Reply:
Fixed, thanks for the review. Out of curiosity, I saw that the Arr::wrap function doesn't cast to array. What is the benefit of that?

Reply:
It does not apply to the use case in this PR, but a `stdClass` is castable to an array, while `Arr::wrap()` allows it to be used as the single element for an array.